webservice/src/main/java/com/hortonworks/registries/webservice/RegistryApplication.java

```diff
@@ -69,6 +72,21 @@ public void run(RegistryConfiguration registryConfiguration, Environment environ
         if (registryConfiguration.isEnableCors()) {
             enableCORS(environment);
         }
+
+        List<ServletFilterConfiguration> servletFilterConfigurations = registryConfiguration.getServletFilters();
```

Comment:
@priyank5485 let's put the code in lines 76-89 in their own method and call it something along the lines **addServletFilters** or **registerServletFilters**, following the pattern of small cohesive methods already existing in this **run** method. It will make the code much cleaner, modular, and easier to read.